eclipse/tests/org.switchyard.tools.m2e.tests/src/org/switchyard/tools/m2e/tests/AbstractSwitchYardTest.java

```diff
@@ -32,7 +35,7 @@
      * 
      * @throws Exception
      */
-    protected void waitForJobs() throws Exception {
+    protected void waitForJobsOld() throws Exception {
```

Comment:
Not still used, but would like to keep it there for historical reasons. That said, I'll adjust it a bit and maybe that's just me not liking getting rid of things.